pkgs/development/python-modules/python-fluent/fluent-syntax.nix

```diff
@@ -0,0 +1,42 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  setuptools,
+  typing-extensions,
+}:
+
+let
+  version = "0.19.0";
+
+  src = fetchFromGitHub {
+    owner = "projectfluent";
+    repo = "python-fluent";
+    rev = "fluent.syntax@${version}";
+    hash = "sha256-nULngwBG/ebICRDi6HMHBdT+r/oq6tbDL7C1iMZpMsA=";
+  };
+in
+buildPythonPackage {
+  pname = "fluent-syntax";
+  inherit version;
+  pyproject = true;
+
+  inherit src;
+  sourceRoot = "${src.name}/fluent.syntax";
+
+  dependencies = [
+    setuptools
+    typing-extensions
+  ];
+
+  pythonImportsCheck = [ "fluent.syntax" ];
+
```

Comment:
Could you enable tests with `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook

Reply:
Oh, I totally missed the tests. Thanks, I've updated this.